hello.ahk

```diff
@@ -15,103 +15,57 @@ Global VarL := ""
 DetectHiddenWindows, On ; BEcause most AHK script have their window hidden
 SetTitleMatchMode, 2
 
-GroupAdd, SuperMemo, ahk_class TBrowser ;Browser
```

Comment:
I suggest that you try to always group the codes to functions for easier reading and debugging. Example: function_initial_setup()